### PR TITLE
(linters errors fixed locally) Milestone 4 Javascript module for the actual calculator.

### PR DESCRIPTION
**Pull request by Abdoulaye-Thespy**
In this milestone, we create two modules (calculate and operate) to perform different operations with our calculator.
(linters errors are fixed locally)

### CODE REVIEW

File: src/logic/calculate.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import operate from './operate';
+
+function calculate(calculatorData, buttonName) {
+  let { total, next, operation } = calculatorData;
+  const operators = ['+', 'x', '-', '÷'];
```
Comment (wathigo) [inline src/logic/calculate.js:5]: **REQUIRED**
* Operate switches operation and uses plus, minus, times, div and modifier (turning the number into a percentage based number, e.g. 95 % == .95) functions ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone4/performOperate.png))
Let's include the modifier operator.

**Review by wathigo — CHANGES_REQUESTED** [review]
### Changes Required :heavy_multiplication_x: 
Great job on this milestone.
Almost all requirements have been met.
Let's fix a few issues before we proceed to the next milestone.

**REQUIRED**
* Let's make sure all the linter check passes.
* Let's fix the issue pointed on the comment :point_down:.

**After making the changes, submit another request for review.**

**Happy Coding :technologist: 
Simon Wathigo.**

File: src/logic/calculate.js (hunk 1 above)
Comment (Abdoulaye-Thespy) [reply src/logic/calculate.js:5]: linters errors fixed locally

File: src/logic/operate.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    case '-':
+      return one.minus(two).toString();
+    case 'x':
+      return one.times(two).toString();
+    case '÷':
+      try {
+        return one.div(two).toString();
+      } catch (error) {
+        return 'NaN';
+      }
```
Comment (wathigo) [inline src/logic/operate.js:21]: **REQUIRED**
The modifier operator has been added but the operation is not yet supported.

**Review by wathigo — CHANGES_REQUESTED** [review]
### Changes Required
We have added the modifier operator as part of the array.
The functionality is still not supported inside our `operate.js`. See :point_down: 

**After making the changes, submit another request for review.**

**Happy Coding :technologist: 
Simon Wathigo.**

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 
Nice work implement the required change by the previous TSE. We however, need to fix the linter error before we can merge. 
Linters allow our code to be error free and formatted in a professional way. It's important to fix all errors.

### Martin Afani

**Comment by Abdoulaye-Thespy** [comment]
The project is error free, Every single linters errors have been fixed locally.

**Review by bellom — APPROVED** [review]
## Hi @Abdoulaye-Thespy  :clap: 

**Great work making all the necessary corrections from the previous review.**

# Status: Approved 👍 

**Quick one:** **It would be nice to look into the linter's error together, we could jump on a call. I can see you have been this issue since milestone 1.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**